opentelemetry-sdk/src/export/trace.rs

```diff
@@ -16,7 +16,7 @@ pub type ExportResult = Result<(), TraceError>;
 /// The goal of the interface is to minimize burden of implementation for
 /// protocol-dependent telemetry exporters. The protocol exporter is expected to
 /// be primarily a simple telemetry data encoder and transmitter.
-pub trait SpanExporter: Send + Debug {
+pub trait SpanExporter: Send + Sync + Debug {
```

Comment:
Please add an entry to changelog.md file.

Reply:
Done